src/modules/BootStorage.rb

```diff
@@ -192,8 +192,10 @@ def detect_disks
       if Mode.config
         Mode.SetMode("normal")
         log.info "Initialize libstorage in readonly mode" # bnc#942360
+        # Set StorageDevices flag disks_valid to true. So InitLibstorage
+        # can scan valid disks. (bnc#1046738)
```

Comment:
Same than above I would reference both bugs here so merging to SP3/master don't kill the old number (the original bug one).